easybuild/easyconfigs/p/Paraview/Paraview-4.2.0-intel-2014b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'Paraview'
+version = '4.2.0'
+
+homepage = "http://www.paraview.org"
+description = "Paraview is a scientific parallel visualizer"
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+toolchainopts = {'usempi': False, 'pic': True}
+
+download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
+source_urls = ['http://www.paraview.org/paraview-downloads/%s' % download_suffix]
+sources = ["ParaView-v%(version)s-source.tar.gz"]
+
+patches = ['missingheader-%(version)s.patch']
+
+dependencies = [
+    ('Qt','4.8.6'),
+    ('Mesa', '10.4.5', '-Python-2.7.8'),
```

Comment:
I'm OK with abandoning this in favor of a 2015a or 2015b effort (the latter implies also redoing Mesa with 2015b...)

It's important to keep PRs somewhat minimal; this PR already throws a whole bunch of things together (see the libdrm and libxml2 changes).

Paraview should be handled in a separate PR for example, not involving OpenFOAM

if remarks are made to an easyconfig that should be fixed in all existing easyconfigs for that software (cfr. libdrm) handle that in a separate PR too (to avoid blocking this PR for it)